core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -226,13 +227,13 @@ private void populateValues(HttpServletRequest request) {
 
   private void setUfsAttributes(HttpServletRequest request) {
     FileSystemMaster fsMaster = mMasterProcess.getMaster(FileSystemMaster.class);
-    Map<String, MountPointInfo> mountPoints = fsMaster.getMountTable();
-    MountPointInfo mountInfo = mountPoints.get(MountTable.ROOT);
-    if (mountInfo == null) {
-      LOG.error("Missing root mount info");
+    MountPointInfo mountInfo;
+    try {
+      mountInfo = fsMaster.getMountPointInfo(new AlluxioURI(MountTable.ROOT));
+    } catch (Throwable e) {
+      LOG.error("Unable to get mount point information of Alluxio root", e);
       return;
```

Comment:
We need to set "diskCapacity", "diskUsedCapacity", etc. to reasonable values here

Reply:
Set them to "Unknown", thanks

Reply:
@aaudiber Remove the set "UNKNOWN" values because the values will show as `UNKNOWN` by default in web UI if we do not provide values. I think `UNKNOWN` is reasonable here. What other reasonable values in your mind that we should set? Thanks

Reply:
How are you testing this? I tried testing locally and get a stack trace in the UI if we return here

Reply:
Oh, I test it in a wrong way. Now I add back the set attributes to "UNKNOWN" and push the
```
    StorageTierInfo[] infos = generateOrderedStorageTierInfo();
    request.setAttribute("storageTierInfos", infos);
```
out of the `setUfsAttributes` function because it is only related to block master and Alluxio storage.
@aaudiber PTAL, thanks!